<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUvVisImportDialog</class>
 <widget class="QDialog" name="SettingsUvVisImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Device Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QGroupBox" name="infoGroupBox">
     <property name="title">
      <string>Device Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="deviceNameLabel">
        <property name="text">
         <string notr="true">Printalyzer Densitometer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deviceVersionLabel">
        <property name="text">
         <string notr="true">Version:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deviceUidLabel">
        <property name="text">
         <string notr="true">Device UID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="exportDateLabel">
        <property name="text">
         <string notr="true">Export Date:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="reflGroupBox">
         <property name="title">
          <string>VIS Reflection Calibration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,3">
            <item row="2" column="2">
             <widget class="QLabel" name="reflCalHiReadingLabel">
              <property name="text">
               <string notr="true">5.000000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="reflCalLoReadingLabel">
              <property name="text">
               <string notr="true">110.000000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="reflCalHiDensityLabel">
              <property name="text">
               <string notr="true">1.50</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="reflCalLoDensityLabel">
              <property name="text">
               <string notr="true">0.08</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_26">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>CAL-HI</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_25">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>CAL-LO</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Density</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>Light Reading</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="importReflCheckBox">
            <property name="text">
             <string>Import VIS reflection</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tranGroupBox">
         <property name="title">
          <string>VIS Transmission Calibration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,3">
            <item row="2" column="2">
             <widget class="QLabel" name="tranCalHiReadingLabel">
              <property name="text">
               <string notr="true">0.300000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="tranCalLoReadingLabel">
              <property name="text">
               <string notr="true">275.000000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="tranCalHiDensityLabel">
              <property name="text">
               <string notr="true">2.90</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="tranCalLoDensityLabel">
              <property name="text">
               <string notr="true">0.00</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_37">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>CAL-HI</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_38">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>CAL-LO</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_39">
              <property name="text">
               <string>Density</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_40">
              <property name="text">
               <string>Light Reading</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="importTranCheckBox">
            <property name="text">
             <string>Import VIS transmission</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="uvTranGroupBox">
         <property name="title">
          <string>UV Transmission Calibration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,3">
            <item row="2" column="2">
             <widget class="QLabel" name="tranUvCalHiReadingLabel">
              <property name="text">
               <string notr="true">0.300000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="tranUvCalLoReadingLabel">
              <property name="text">
               <string notr="true">275.000000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="tranUvCalHiDensityLabel">
              <property name="text">
               <string notr="true">2.90</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="tranUvCalLoDensityLabel">
              <property name="text">
               <string notr="true">0.00</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_41">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>CAL-HI</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_42">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>CAL-LO</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_43">
              <property name="text">
               <string>Density</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_44">
              <property name="text">
               <string>Light Reading</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="importUvTranCheckBox">
            <property name="text">
             <string>Import UV transmission</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="gainGroupBox">
         <property name="title">
          <string>Gain Calibration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="gainTableWidget">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <row>
             <property name="text">
              <string>0.5x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>16x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>32x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>64x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>128x</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>256x</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="importGainCheckBox">
            <property name="text">
             <string>Import gain</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tempGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Temperature Calibration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QTableWidget" name="tempTableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
            </property>
            <row>
             <property name="text">
              <string>B0</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>B1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>B2</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>VIS</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>UV</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="importTempCheckBox">
            <property name="text">
             <string>Import temperature</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
